server/routes/incidentDetails/index.ts

```diff
@@ -0,0 +1,27 @@
+import express, { RequestHandler, Router } from 'express'
+import asyncMiddleware from '../../middleware/asyncMiddleware'
+
+import IncidentDetailsRoutes from './incidentDetails'
+
+import PlaceOnReportService from '../../services/placeOnReportService'
+import LocationService from '../../services/locationService'
+
+export default function prisonerIncidentStatementsRoutes({
```

Comment:
Minor naming issue - details not statement

Reply:
good catch!